week-1/interpret/percentage-change.js

```diff
@@ -9,14 +9,42 @@ const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+/*
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+There are three function calls in this code:
+
+carPrice.replaceAll(",", "") on line 4
```

Comment:
You're almost right here - you're heading in the right direction! You've said that we call `replaceAll` on line 4 and line 5. And we also call `Number` on lines 4 and lines 5 too. So how many function calls in total?